net/miniupnpd/patches/201-change-default-chain-rule-to-accept.patch

```diff
@@ -0,0 +1,13 @@
+--- a/netfilter_nft/scripts/nft_init.sh
```

Comment:
1. Who is the author of this change?
2. Why is this change required?
- There is a missing commit description!
3. Was this discussed with upstream?
- Is there any pending pull request? If not, submit it there first.

Reply:
1. I do this change(Author: ZiMing Mo <msylgj@immortalwrt.org>)
2. We can find the reason in https://github.com/miniupnp/miniupnp/blob/master/miniupnpd/netfilter_nft/README.md
```
Custom Chains
NFTables is very flexible but it comes with some restrictions because of that. If there is a second filter chain than all packets that were passed before with the miniupnpd chain will be reevaluated. This also means that if the chain is a drop chain you loose the packets. In that case you really want to use a custom chain and jump to it in your filter chain. miniupnpd should save all accept rules in that custom chain. For NAT it is the same, a second chain will also evaluate the packets again and therefore it is possible that a second SNAT or DNAT is performed.

The following is used in miniupnpd for a table setup but it can be customized
```
There have a filter chain in table inet fw4(created by firewall4), it's a drop chain. if the filter chain of miniupnpd is also a drop chain, all packets will be loose(In my testing, it happend). So i do the change for miniupnpd work on firewall4.
3. Based on the description above, this change doesn't need an upstream PR or discuss, in my mind.

Reply:
All of this, should be included in the patch itself. Then it is a question, why it should not be a default for miniupnpd and we do need to modify this in OpenWrt.